core/schema/module.go

```diff
@@ -183,6 +183,10 @@ func (s *moduleSchema) Install() {
 	dagql.Fields[*core.GitModuleSource]{
 		dagql.Func("htmlURL", s.gitModuleSourceHTMLURL).
 			Doc(`The URL to the source's git repo in a web browser`),
+		dagql.Func("cloneURL", s.gitModuleSourceCloneURL).
+			View(BeforeVersion("v0.13.0")).
+			Doc(`The URL to clone the root of the git repo from`).
+			Deprecated(`Use CloneRef instead. CloneRef supports both URL-style and SCP-like SSH references`),
```

Comment:
The backticks in the reason message are understood by codegen to format the field name to the language's convention:
```suggestion
			Deprecated("Use `cloneRef` instead. `cloneRef` supports both URL-style and SCP-like SSH references"),
```